core/code/map.js

```diff
@@ -289,7 +289,7 @@ window.setupMap = function () {
       pos = {center: [0, 0], zoom: 1};
       map.locate({setView: true});
     }
-    map.setView(pos.center, pos.zoom, {reset: true});
+    map.setView(pos.center, pos.zoom);
```

Comment:
good point
curiously, it's still used in the leaflet code to force _resetView call.

Reply:
looks like a now undocumented leaflet feature
reverted in a8e5ee9